infra.yaml

```diff
@@ -265,7 +265,12 @@ resources:
   host:
     type: OS::Nova::Server
     properties:
-      name: {get_param: hostname}
+      name:
+        str_replace:
+          template: "HOST.DOMAIN"
+          params:
+            HOST: {get_param: hostname}
+            DOMAIN: {get_param: domain_name}
```

Comment:
This might be a problem - the reason I removed domain from node names was that it doesn't play well with external DNS -> then it creates DNS records like "infra.example.com.example.com". What is the reason for isntancename==FQDN in this case?